resources/views/dashboard/admin/store/index.blade.php

```diff
@@ -0,0 +1,60 @@
+@extends('layouts.dashboard')
+
+@section('title')
+Merch Store Management
+@endsection
+
+@section('content')
+@include('inc.header', ['title' => 'ZTL Merch Store'])
+
+<div class="container">
+    <a href="/dashboard/admin/store/new" class="btn btn-primary">Create New Item</a>
+    <br><br>
+    @if(count($merch_items) > 0)
+        <table class="table table-outline">
+            <thead>
+                <tr>
+                    <th scope="col" colspan="2">Item</th>
+                    <th scope="col">Description</th>
+                    <th scope="col">Price</th>
+                    <th scope="col">Actions</th>
+                </tr>
+                @foreach($merch_items as $merch_item)
+                    <tr>
+                        <td>
+                            @if($merch_item->image)
+                                <img src="{{ url('/storage/store/'.$merch_item->image) }}" width="300px"></img>
+                            @else
+                                <img src="/photos/No_image_available.svg" width="100px"></img>
+                            @endif
+                        </td>
+                        <td>
+                            @if(!is_null($merch_item->flag))
+                                <span class="badge badge-{{ $merch_item->displayFlag()->badge }}">{{ $merch_item->displayFlag()->text }}</span><br>
+                            @endif
+                            <b>{{ $merch_item->title }}</b>
+                        </td>
+                        <td>{{ $merch_item->description }}</td>
+                        <td>${{ $merch_item->price }} USD</td>
+                        <td>
+                            <div class="row">
```

Comment:
Can this be changed to a standard bootstrap button group? Right now the buttons overlap and the "Edit" button isn't clickable (tested on Firefox 150).
![image](https://github.com/user-attachments/assets/85caec12-17b6-4b15-aeeb-02d25cdd34a3)